arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts

```diff
@@ -0,0 +1,76 @@
+/*
+ * vc4-kms-v3d-pi4-overlay.dts
+ */
+
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/clock/bcm2835.h>
+
+&{/chosen} {
```

Comment:
The overlay checking tools don't yet understand the new overlay syntax, and this overlay is missing a compatible string. You'll need to use the old syntax for now if you want it accepted.